mujoco_warp/_src/types.py

```diff
@@ -535,6 +535,8 @@ class Option:
     graph_conditional: flag to use cuda graph conditional, should be False when JAX is used
     sdf_initpoints: number of starting points for gradient descent
     sdf_iterations: max number of iterations for gradient descent
+    run_collision_detection: flag to run collision detection (if the collision detection is
```

Comment:
nit: maybe make it clearer to the reader why this exists, something like:

```
    run_collision_detection: if False, skips collision detection and allows user-populated
                             contacts during the physics step  (as opposed to `DisableBit.CONTACT`
                             which explicitly zeros out the contacts at each step)
```

Something like that.